frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -298,8 +298,16 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
           isInline
           className="co-alert"
           variant="danger"
-          title="Namespace does not support install modes for this Operator"
-        />
+          title="Namespace does not support installation mode"
+        >
+          {`The operator group in the '${selectedTargetNamespace}' namespace does not support the
+          ${
+            selectedInstallMode === InstallModeType.InstallModeTypeAllNamespaces
+              ? 'global'
+              : 'single namespace'
+          }
+          installation mode.`}
```

Comment:
No reason for this to be a string, it can be markup.

```suggestion
          The operator group in the {selectedTargetNamespace} namespace does not support the
          {
            selectedInstallMode === InstallModeType.InstallModeTypeAllNamespaces
              ? 'global'
              : 'single namespace'
          }
          installation mode.
```